tests/utils/test_client_management_run.py

```diff
@@ -0,0 +1,69 @@
+import tempfile
+from subprocess import run, STDOUT, PIPE
+
+import pytest
+
+import oic.utils.client_management
+from oic.utils.client_management import CDB, pack_redirect_uri
+
+CLI_PATH = oic.utils.client_management.__file__
+CLI_INVOCATION = 'python {} '.format(CLI_PATH)
+
+
+@pytest.fixture
+def db_file_path():
+    db_file = tempfile.NamedTemporaryFile(delete=True)
+    db_file.close()
+    return db_file.name
+
+
+class TestClientManagementRun(object):
```

Comment:
How this relates to the existing `tests.client_management` test suite; this tests the command-line invocation of the utility, as opposed to the CDB class it wraps.